gtk4/src/auto/application.rs

```diff
@@ -59,14 +59,6 @@ impl ApplicationBuilder {
         }
     }
 
-    pub fn action_group(self, action_group: &impl IsA<gio::ActionGroup>) -> Self {
```

Comment:
No idea what happened

```xml
      <property name="action-group" writable="1" construct-only="1" transfer-ownership="none">
```

Reply:
that is expected, the property comes from GApplication has been deprecated since 2.38 which we don't support anymore. It shouldn't have been there at all

Reply:
It was added back, no idea why.

Reply:
Update the gir-files submodule?